backend/src/Controllers/controllerUtils.ts

```diff
@@ -38,4 +38,8 @@ export function expectPermission(roles:roles[],permission:permission):any {
         if (!roles.some( (role) => permission.includes(role))){
             throw new Unauthorized("Does not have permission")
       }
+}
+
+export function hashString(inputString: string) {
+    return createHash('sha1').update(inputString).digest('hex')
```

Comment:
sha256 might be a better option

sounds like sha1 is still pretty secure overall, but if we're committing to one I figure we may as well go with the stronger option

https://www.keycdn.com/support/sha1-vs-sha256#:~:text=SHA1%20is%20no%20longer%20considered,should%20also%20switch%20to%20SHA256.

https://crypto.stackexchange.com/questions/48289/how-secure-is-sha1-what-are-the-chances-of-a-real-exploit